web/angular-wallet/src/app/login/login.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from '@angular/core';
+// import { AccountService } from '../../lib/services';
+// import { CreateService } from '../dashboard/setup/account/create.service';
+// import { NotifierService } from 'angular-notifier';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss'],
+})
+export class LoginComponent implements OnInit {
+
+  constructor(
+    // private accountService: AccountService,
+    // private createService: CreateService,
+    // private notificationService: NotifierService,
+    private router: Router) {}
+
+  method = 'active';
+  version = 'v0.0.1'; // FIXME: bump version
```

Comment:
This should probably come from package.json

Reply:
Yeah, thought the same. In nodejs using package.json is straightforward. In web apps we usually don't deploy package.json, so we would need may a pre-built-script  that generates a version.ts file, like here: https://medium.com/@amcdnl/version-stamping-your-app-with-the-angular-cli-d563284bb94d